src/services/searchSkylink.js

```diff
@@ -107,11 +93,27 @@ export default async function searchSkylink(input) {
 
   if (input.match(IPFS_CID_MATCHER)) {
     const { groups } = input.match(IPFS_CID_MATCHER);
-    const skylink = migrateIpfsToSkylink(groups.cid);
+    const skylink = await migrateIpfsToSkylink(groups.cid);
```

Comment:
I guess it's either adding `await`, or just `return migrateIpfsToSkylink(groups.cid);` here. It's redundant to set `skylink` to a promise and then check it because it'll always be evaluated as true and returned.